Lib/test/list_tests.py

```diff
@@ -191,6 +191,14 @@ def test_setslice(self):
 
         self.assertRaises(TypeError, a.__setitem__)
 
+    def test_slice_assign_iterator(self):
+        x = self.type2test(range(5))
+        x[0:3] = iter(self.type2test(reversed(range(3))))
```

Comment:
Is there a difference between `iter(self.type2test(reversed(range(3))))` and `reversed(range(3))`?

Reply:
Good point, test can be further simplified 👍